Source/WebKit/NetworkProcess/storage/SessionStorageManager.cpp

```diff
@@ -62,6 +62,16 @@ void SessionStorageManager::connectionClosed(IPC::Connection::UniqueID connectio
         storageArea->removeListener(connection);
 }
 
+void SessionStorageManager::removeNamespace(StorageNamespaceIdentifier namespaceIdentifier)
+{
+    auto identifier = m_storageAreasByNamespace.take(namespaceIdentifier);
+    if (!identifier.isValid())
+        return;
+
+    auto storageArea = m_storageAreas.take(identifier);
```

Comment:
Do we need to take, wouldn't remove() work?

Reply:
Yes, remove should work; will update.